library/app/src/main/res/values/strings.xml

```diff
@@ -15,5 +15,5 @@
     <string name="button_more">More</string>
     <string name="button_less">Less</string>
     <string name="title_activity_splash_screen">SplashScreenActivity</string>
-    <string name="content_extraction">यह प्रक्रिया केवल एक बार की जाएगी</string>
+    <string name="content_extraction">यह प्रक्रिया केवल एक बार की जाएगी इसमें १० मिनट तक का समय लग सकता है।</string>
```

Comment:
Add a khadi matra after jayegi and a space after that. Just copy paste entire thing from Chimple or OneBillion strings.xml